File: Selenium/Activities/Activity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox()  as ffDriver :
    ffDriver.get("https://v1.training-support.net/selenium/login-form")
    print("title of web pageis : ",ffDriver.title)
    username = ffDriver.find_element(By.XPATH, "//input[@id='username']")
    password = ffDriver.find_element(By.XPATH, "//input[@id='password']")
    username.send_keys("admin")
    password.send_keys("password")
    ffDriver.find_element(By.XPATH,"//button[text()='Log in']").click()

    txt = ffDriver.find_element(By.XPATH,"//div[@id='action-confirmation']");
    print(txt.text)
    ffDriver.quit()

File: Selenium/Activities/Activity3.java
package activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/login-form");

		System.out.println("The Title of the page is : " + driver.getTitle());

		driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
		driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
		driver.findElement(By.xpath("//button[text()='Log in']")).click();

		String txt = driver.findElement(By.xpath("//div[@id='action-confirmation']")).getText();
		System.out.println("Login message: " + txt);

		driver.close();
	}

}
